src/modules/browse.js

```diff
@@ -118,6 +118,28 @@ function createBrowseUrlFromIDs(itemIds, parameters, userParameters, options) {
   return `${serviceUrl}/browse/items?${queryString}`;
 }
 
+// Create request headers using supplied options and user parameters
+function createHeaders(options, userParameters) {
```

Comment:
Thoughts on calling this something like `applyHeaders` instead?

Reply:
The ones that are doing the same thing for `queryParams` are named `create`. I think this is doing the same thing for `headers` and it is returning an object instead of applying anything. I think `create` works better but we can rename to `apply` if you want